src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,53 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+
+    private static final String PACKAGE_NAME = "mate.academy";
+    private static final Injector INJECTOR = Injector.getInstance(PACKAGE_NAME);
+    private static final String TITLE = "Fast and Furious";
+    private static final String DESCRIPTION_OF_MOVIE = "An action film about street racing, "
+            + "heists, and spies.";
+    private static final long MOVIE_ID_ONE = 1L;
```

Comment:
The `MOVIE_ID_ONE` is hardcoded as `1L`, which may not correspond to the actual ID of the `fastAndFurious` movie object. Consider using `fastAndFurious.getId()` to ensure the correct ID is used.